internal/servers/controller/handlers/accounts/account_service.go

```diff
@@ -57,10 +77,20 @@ func (s Service) ListAccounts(ctx context.Context, req *pbs.ListAccountsRequest)
 	if err != nil {
 		return nil, err
 	}
+	finalItems := make([]*pb.Account, 0, len(ul))
+	resource := &perms.Resource{
```

Comment:
Might be good to name this `res` so it doesn't collide with package `resource`, was a little confused at first when seeing the use of `resource.Account` when setting type or `resource`

Reply:
Good call, fixed.